waku/whisper/whispernodes.nim

```diff
@@ -0,0 +1,67 @@
+const
```

Comment:
Slightly off-topic: Should we have a CI workflow that autogenerates this file weekly?

Reply:
We try not to meddle to much in the "stable" (and unused) v1 code, but, more importantly, `fleets.status.im` where this comes from is not supposed to be used as a source of truth - it was meant to be a tool of convenience. For Waku v2 we encode lists like these in a proper, signed merkle tree structure hosted on a domain server somewhere.